lib/location/Placemark.ts

```diff
@@ -44,4 +44,13 @@ export const placemarkToFormattedAddress = (placemark: Placemark) => {
   return formattedAddress === US_COUNTRY_CODE ? undefined : formattedAddress
 }
 
+/**
+ * Formats a `Placemark` instance into a short address (name, city, region/state)
+ *
+ * At the moment, this only outputs US style addresses.
+ */
+export const placemarkToAbbreviatedAddress = (placemark: Placemark) => {
+  return `${placemark.name}, ${placemark.city}, ${placemark.region}`
```

Comment:
In case any of these are null we should have fallbacks @mhayes853

Reply:
Maybe something like:
- No Name -> Just render city and region
- No Region or City -> Render "Unknown Location"